TRANSLATING.md

```diff
@@ -0,0 +1,72 @@
+# Translating Guide
+
+## Introduction
+
+It is possible to translate the website to other languages and anyone is very welcome.
+
+Note that we're taking translations very seriously.
+They should receive as much love as the English version.
+
+For example, Google Translate is already built into Google Chrome.
+Don't bother using software to create translations of the docs.
+If we create a translation of the docs for a new language, then because we want high quality translations written by native speakers or near-native speakers.
+Examples and real-world comparisons must be replaced by equivalents in the culture of the respective country.
+A good translation is hard and it takes time. We don't want partial or outdated translations.
+
+## The Translation Workflow
+
+Translating the grammY docs into a language happens in two stages: the Active Stage and the Maintenance Stage.
+
+When a language is first started, no translations are available.
+Hence, the complete website must be translated actively (Active Stage).
+This requires an insane amount of work and a fair bit of dedication (as you all know).
+Respect to everyone who makes it through this.
+
+The language will then enter Maintenance Stage.
+From here on, all that needs to be done is to keep notifications on for new pull requests, and sync the translations whenever an article gets updated.
+
+Here is some more information about each stage works.
+
+### Active Stage
+
+A translation to another language must be 100% completed before it gets published.
+We do not accept partial translations.
+All menus, links, articles, comments, and everything translatable must be localised before they can go live on the website.
+
+It would be a pain in the butt to do everything in a single pull request, so we developed a simple branching model which will make this easy.
```

Comment:
```suggestion
It would be very painful to do everything in a single pull request, so we developed a simple branching model which will make this easy.
```